kediatr-core/src/main/kotlin/com/trendyol/kediatr/CommandBusBuilder.kt

```diff
@@ -1,6 +1,8 @@
 package com.trendyol.kediatr
 
-class CommandBusBuilder(private val clazzOfAnyHandler: Class<*>) {
+class CommandBusBuilder(
+    private val dependencyProvider: DependencyProvider
```

Comment:
Not important though, if you wish, you may take a Registry instance here. Why did I thought like that? The library provides a Registry interface to the clients for the possiblity of a custom Registry but i think the clients are not able to use a custom Registry when building a CommandBus.